core/playwright/helpers.ts

```diff
@@ -102,6 +102,6 @@ export const PubFieldsOfEachType = Object.fromEntries(
 
 export const waitForBaseCommunityPage = async (page: Page, communitySlug?: string) => {
 	await page.waitForURL(new RegExp(`.*/c/${communitySlug ?? ".*"}/stages.*`), {
-		timeout: 5_000,
+		timeout: 10_000,
```

Comment:
ran into too many issues when running e2e in dev mode with this small timeout

Reply:
Oops, I think I forgot to undo this change in one of my recent commits.